### ISSUE TITLE
'NoneType' object has no attribute 'decode'

### ISSUE THREAD

**Issue by pilate** [mandiant/flare-floss#213]
d69e6a6925f9229d5840b6eade99ab4a

```
Traceback (most recent call last):
  File "floss/main.py", line 600, in main
    vw = viv_utils.getWorkspace(sample_file_path, should_save=options.save_workspace)
  File "/usr/local/lib/python2.7/dist-packages/viv_utils-0.3.5-py2.7.egg/viv_utils/__init__.py", line 32, in getWorkspace
    vw.loadFromFile(fp)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/__init__.py", line 2134, in loadFromFile
    fname = mod.parseFile(self, filename)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/parsers/pe.py", line 30, in parseFile
    return loadPeIntoWorkspace(vw, pe, filename)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/vivisect/parsers/pe.py", line 113, in loadPeIntoWorkspace
    vs = pe.getVS_VERSIONINFO()
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/PE/__init__.py", line 550, in getVS_VERSIONINFO
    vbytes = self.readResource(RT_VERSION, 1)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/PE/__init__.py", line 513, in readResource
    rsdef = self.getResourceDef(rtype, name_id)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/PE/__init__.py", line 495, in getResourceDef
    return self.ResourceRoot.getResourceDef(rtype, name_id)
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/PE/__init__.py", line 1079, in __getattr__
    self.parseResources()
  File "/usr/local/lib/python2.7/dist-packages/vivisect-0.0.0-py2.7.egg/PE/__init__.py", line 595, in parseResources
    name_id = self.readAtRva(namerva + 2, namelen * 2).decode('utf-16le', 'ignore')
AttributeError: 'NoneType' object has no attribute 'decode'
```

```
ERROR:floss:Vivisect failed to load the input file: 'NoneType' object has no attribute 'decode'
```

**Comment by mr-tz** [comment]
currently getting:
`ERROR:floss:Failed to print vivisect analysis information: too many values to unpack (expected 2)}`

**Comment by williballenthin** [comment]
this "too many values to unpack" bug is fixed in #408. i introduced it during the last PR, sorry.

**Comment by mr-tz** [comment]
fixed in current master
